zeppelin-web/src/app/notebook/paragraph/paragraph-control.html

```diff
@@ -133,6 +133,12 @@
             <span class="shortcut-keys">Ctrl+{{ isMac ? 'Option' : 'Alt'}}+m</span></a>
         </li>
         <li>
+          <a  ng-click="runSubsequentParagraphs(parentNote,paragraph)"
+              ng-show="paragraph.status!='RUNNING' && paragraph.status!='PENDING' && paragraph.config.enabled">
+            <span class="icon-control-end shortcut-icon" tooltip="Run this paragraph and the next ones"></span>Run with next ones
```

Comment:
Not sure other users can notice "Run with next ones" means "Run with following all paragraphs". Can we use more intuitive name for this?